js/build.js

```diff
@@ -21,11 +24,36 @@ Job.prototype.toHtml = function() {
   return $newJob;
 };
 
+// INSTANTIATING JOB EXP OBJECTS FROM SOURCE.JS AND PUSHING INTO ARRAY
 expEntries.forEach(function(options) {
   entries.push(new Job(options));
 });
 
+// APPENDING JQUERY OBJECTS TO HTML PAGE
 entries.forEach(function(currentEntry) {
   $('#experience').append(currentEntry.toHtml());
-  console.log(currentEntry);
+});
+
+// CONSTRUCTOR FUNCTION FOR PROJECTS
+function ProjectEntry (opts) {
+  this.projName = opts.projName;
+  this.projSkills = opts.projSkills;
+  this.projDesc = opts.projDesc;
+};
+
+// COMPILING PROJECT OBJECTS WITH HANDLEBARS
+ProjectEntry.prototype.toHtml = function() {
```

Comment:
I like that you combined this on the same script file with the JobEntry one, and gave them consistent names. This is something I may adapt as well to help modularize the scripts (since they do the same thing, but with two different objects).